addons/mail/static/src/components/chatter/chatter_tests.js

```diff
@@ -452,6 +461,160 @@ QUnit.test('do not post message with "Enter" keyboard shortcut', async function
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+
+    assert.containsOnce(
+        document.body,
+        '.o_ComposerSuggestedRecipientList',
+        "Should display a list of suggested recipients after opening the composer from 'Send message' button"
+    );
+    assert.containsN(
+        document.body,
+        '.o_ComposerSuggestedRecipient',
+        3,
+        "There should be 3 suggested recipients"
+    );
+
+    assert.containsNone(
+        document.body,
+        '.o_ComposerSuggestedRecipientList_showMore',
+        "suggested recipient list should not containt a \"show more\" button."
+    );
+
+});
+
+QUnit.test("suggested recipients reason should be displayed as title", async function (assert) {
+    assert.expect(1);
+
+    this.data['res.fake'].records.push({
+        id: 10,
+        email_cc: "john@test.be",
+    });
+    await this.start ();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+
+    const title = document.querySelector('.o_ComposerSuggestedRecipient:not([data-partner-id])').getAttribute('title');
+    assert.strictEqual(
+        title,
+        "Add as recipient and follower (reason: CC email)",
+        "reason must be present in the suggested recipient title",
+    );
+});
+
+QUnit.test("suggested recipients should be checked when there is a partner and unchecked if there is no partner", async function (assert) {
+    assert.expect(2);
+
+    this.data['res.fake'].records.push({
+        id: 10,
+        email_cc: "john@test.be",
+    });
+    await this.start ();
+    const chatter = this.env.models['mail.chatter'].create({
+        threadId: 10,
+        threadModel: 'res.fake',
+    });
+    await this.createChatterComponent({ chatter });
+
+    await afterNextRender(() =>
+        document.querySelector(`.o_ChatterTopbar_buttonSendMessage`).click()
+    );
+
+    const checkboxUnchecked = document.querySelector('.o_ComposerSuggestedRecipient:not([data-partner-id]) input[type=checkbox]');
+    assert.notOk(
+        checkboxUnchecked.checked,
+        "suggested recipient without a partner should not be checked by default",
+    );
+    const checkboxChecked = document.querySelector('.o_ComposerSuggestedRecipient[data-partner-id] input[type=checkbox]');
+    assert.ok(
+        checkboxChecked.checked,
+        "suggested recipient with a partner should be checked by default.",
+    );
+});
+
+QUnit.test("suggested recipients list should have a show more and show less button", async function (assert) {
```

Comment:
When? Doesn't seem to apply everytime...
This seems dumb, because you know it depends on amount of suggested recipient (magic values are 3 & 4), but it's not explicit here so no good.